src/SwedbankPay.Sdk/LinkRelation.cs

```diff
@@ -0,0 +1,65 @@
+﻿#region License
+
+// --------------------------------------------------
+// Copyright © Swedbank Pay. All Rights Reserved.
+// 
+// This software is proprietary information of Swedbank Pay.
+// USE IS SUBJECT TO LICENSE TERMS.
+// --------------------------------------------------
+
+#endregion
+
+namespace SwedbankPay.Sdk
+{
+    public sealed class LinkRelation : TypeSafeEnum<LinkRelation, string>
```

Comment:
Why do we need this abstraction layer?

Reply:
Because LinkRelation is used as an Enum, and Operations inherits from Dictionary<LinkRelation, HttpOpertation> 
This is something we changed in discussion with @asbjornu